Sources/CMakeLists.txt

```diff
@@ -112,9 +112,20 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
 	# TODO fix these warnings
 	add_compile_options(-Wno-sign-compare)
 	add_compile_options(-Wno-switch)
-	add_compile_options(-Wno-format-security)
-	add_definitions(-Wno-logical-op-parentheses)
-	MESSAGE(WARNING, "re-enable -Wlogical-op-parentheses some day!")
+	add_compile_options(-Wno-unknown-pragmas)
+	add_compile_options(-Wno-unused-variable)
+	add_compile_options(-Wno-unused-value)
+	add_compile_options(-Wno-reorder)
```

Comment:
I think someone cleaned this reorder warning issues out the other day, so we might not need this one now. I haven't tested personally to see, though.